src/components/card/card.ts

```diff
@@ -0,0 +1,48 @@
+import {Component, View, ViewEncapsulation} from 'angular2/core';
+import {CONST_EXPR, Type} from 'angular2/src/facade/lang';
+
+@Component({
```

Comment:
Could you add a class description comment on each card briefly explaining its role / when you would use it? (it might be helpful to link to images as part of that)